mdtraj/formats/netcdf.py

```diff
@@ -127,21 +128,27 @@ def __init__(self, filename, mode='r', force_overwrite=True):
 
         try:
             # import netcdf4 if it's available
-            netcdf = import_('netCDF4').Dataset
+            import netCDF4
+            netcdf = netCDF4.Dataset
             
             # set input args for netCDF4
             input_args = {'format': 'NETCDF3_64BIT', 'clobber': force_overwrite}
 
         except ImportError:
-            netcdf = import_('scipy.io').netcdf_file
+            
+            import scipy.io
 
-            # warn the user, even though the import above also gives
-            # them a big warning. 
-            warnings.warn('Could not find netCDF4 module. Falling back on '
-                              'scipy implementation, which can be significantly'
-                              'slower than the netCDF4 implementation.')
-        
+            netcdf = scipy.io.netcdf_file
+
+            warning_message = (
+                "Warning: The 'netCDF4' Python module is not installed. MDTraj is using the 'scipy' "
+                "implementation to read and write netCDF files,which can be significantly slower.\n"
+                "For improved performance, consider installing the 'netCDF4' module. See installation instructions at:\n"
+                "https://unidata.github.io/netcdf4-python/#quick-install"
+            )
             
+            print(warning_message, file=sys.stderr)
```

Comment:
How does printing to STDERR differ from using the built-in `warnings` module? I haven't seen this approach before

Reply:
When using `warnings` it prints the full file path for the module and also says the warnings type (`UserWarning`). IMO it's a little distracting from the main message.

I noticed that our previous `import_` function printed the message without the extra info in front, and printing to `stderr` is the approach taken in that function - see https://github.com/mdtraj/mdtraj/blob/d785406885f6c65da61f9992122f17a0a8695c12/mdtraj/utils/delay_import.py#L188

Reply:
If you want to see what the updated warning looks like, I've updated this example notebook: https://github.com/janash/mdtraj-netcdf/blob/main/openmm.ipynb

Reply:
Although, I suppose one could argue that in the `import_` function it's being used with an exception being raised, so might be more appropriate there. 🤔

Reply:
Hmm, I'd prefer everything use `warnings` because it's more configurable and IIUC the standard practice, but it's fine to keep this how it currently is

Reply:
alrighty, `warnings` it is :)